gstreamer.orogen

```diff
@@ -74,6 +74,12 @@ task_context "WebRTCCommonTask", subclasses: "Common" do
     # Configuration of the signalling process
     property "signalling", "/gstreamer/SignallingConfig"
 
+    # The GStreamer object latency
+    property "latency", "base/Time"
+
+    # If the Gstreamer object drops on latency
+    property "drop_on_latency", "bool"
```

Comment:
Another one .... you need to provide a default value. (I would pass true)

For simple types, this can be done in the orogen file (pass as third argument).